package.json

```diff
@@ -138,9 +138,14 @@
                             },
                             "debugRootDir": {
                                 "type": "string",
-                                "description": "If you have a build system, rootDir will point to the build output folder, and this path should point to the actual source folder so that breakpoints can be set in the source files when debugging. In order for this to work, your build process cannot change line offsets between source files and built files, otherwise debugger lines will be out of sync.",
+                                "description": "Depreciated, use sourceDirs, which is an array of source locations",
                                 "default": "${workspaceFolder}"
                             },
+                            "sourceDirs": {
+                                "type": "array",
+                                "description": "If you have a build system, sourceDirs will point to the build output folder, and this path should point to the actual source folder so that breakpoints can be set in the source files when debugging. In order for this to work, your build process cannot change line offsets between source files and built files, otherwise debugger lines will be out of sync.",
```

Comment:
This description doesn't sound correct. should read "**rootDir** will point to the build output folder..."

Also, you should add a brief explanation of how sourceDirs works when you have multiple array entries.